openapi/components/responses/200ServiceInfo.yaml

```diff
@@ -0,0 +1,7 @@
+description: Retrieve info about the DRS service
+content:
+  application/json:
+    schema:
+      allOf:
+        - '$ref': https://raw.githubusercontent.com/ga4gh-discovery/ga4gh-service-info/v1.0.0/service-info.yaml#/components/schemas/Service
+        - '$ref': '../schemas/DrsServiceInfo.yaml'
```

Comment:
What about naming the schema "DrsService"? The "Info' part is not very informative. For example, a user schema would typically be called "User", not "UserInfo".

Reply:
Good point @tschaffter , and also because the base schema is `Service` and not `ServiceInfo`. I'll update